docs/zh-cn/developers/developers_dev.md

```diff
@@ -31,7 +31,8 @@
 | yizhenqiang | Zhenqiang Yi       | Didi         | Committer | +8       |
 | victory    | Victory Cao         | Alibaba      | Committer | +8       |
 | purpleforce | Liandong Chen      | Alibaba      | Committer | +8       |
-| jefflv     | 	Jeff Lv            | Alibaba   |    Committer | +8       |
+| jefflv     | 	Jeff Lv            | Alibaba      | Committer | +8       |
+| tswstarplanet |Taosheng Wei      | NetsUnion    | Committer | +8       |
 
```

Comment:
carryxyh is PPMC now ,would you please help to update it?

Reply:
@carryxyh Excuse me. Can I update your information ?

Reply:
It is ok to update it.  Please also update this page: https://incubator.apache.org/projects/dubbo.html. You need to checkout svn to update it.

Reply:
OK

Reply:
@ralf0131 I don't find dubbo on http://svn.apache.org/repos/asf/
Should dubbo be in other place ?

Reply:
@tswstarplanet 

The documentation can be found at [here](http://incubator.apache.org/guides/website.html)

```
svn co  https://svn.apache.org/repos/asf/incubator/public/trunk
```

Please read the `README.txt` first.

And update `content/projects/dubbo.xml`

When finished, build the html file locally(under `trunk` directory): 

```
ant
```

You can find the generated html files under `target/site/projects/dubbo.html` 

Please confirm everything looks good. And commit only the xml changes, DO NOT commit the html files. HTML files will be automatically generated by jenkins build every day.

Could you please try this and add it to the new committer guide documentation?